mappings/47689e21e411e9acfbeb967bd50127757f03a293206f3cd9e4ced23c545253.json

```diff
@@ -0,0 +1,54 @@
+{
+    "subject": "47689e21e411e9acfbeb967bd50127757f03a293206f3cd9e4ced23c545253",
+    "url": {
+        "sequenceNumber": 0,
+        "value": "https://drive.google.com/file/d/14xftICakqIE5HlKrdRdxwcylhuhHwwoZ/view?usp=drivesdk",
```

Comment:
Hi @siddigsalahaldin, 

The link requires access, so i can't see/approve the content inside. 
Can you please make it public or remove that field? URL value is optional.

Thanks!

Reply:
Done